### PR TITLE
Architecture layers

### PR DESCRIPTION
**Pull request by nieznanysprawiciel**

### CODE REVIEW

File: arch-snapshot/arch.md [hunk 1]
```diff
@@ -125,15 +151,26 @@ monolithic and division of responsibility is not arbitrary.
 The goal of this chapter is to sketch general idea, rather than describe
 specifics.
 ### Network layer
+- Receives message from GSB and propagates them to other Nodes
```
Comment (dopiera) [inline arch-snapshot/arch.md:154]: Not only "receives from", but also "sends to" GSB.

To me GSB is neither on top nor below network - on one hand GSB abstracts the network away, on the other hand GSB forwards traffic to the network. Is there any reason why I should think differently about GSB than about CORBA or DCOM? 

If not, I think we should say "Rust implementation uses our own implementation of CORBA-like system, namely GSB" and omit it in the diagrams.

File: arch-snapshot/arch.md [hunk 2]
```diff
@@ -51,6 +60,20 @@ details and establish a glossary to ensure consistency within the document.
 
 decomposition into layers. responsibility of the layers.
 
+Layers:
```
Comment (dopiera) [inline arch-snapshot/arch.md:63]: I think these are not really layers. This is how components are organized into processes.

File: arch-snapshot/arch.md (hunk 1 above)
Comment (prekucki) [reply arch-snapshot/arch.md:154]: 1. the role is to bridge services between local service buses.
2. yes, it also receives orders from the network and sends them to the local bus.

**Comment by nieznanysprawiciel** [comment]
Obsolete

**Pull request closed without merging**